### PR TITLE
Restyle sidenav

### PR DESCRIPTION
**Pull request by roberthouse54**
#### What are the relevant tickets?
https://github.com/mitodl/micromasters/issues/2564

#### What's this PR do?
This does the style changes required for #2564

#### How should this be manually tested?
Open and close the sidebar. Make sure the links work.

#### Where should the reviewer start?
Check that the css looks like it is properly formatted.

#### Any background context you want to provide?
This only partly completes #2564. The other thing required to complete that issue is to make the sidebar always visible on the page, and to remove the user dropdown in the navbar.

#### Screenshots (if appropriate)
sidenav

### CODE REVIEW

File: static/scss/navbar.scss [hunk 1]
```diff
@@ -243,13 +239,13 @@
...
   .Select-control {
-    border: 1px solid $program_selector_border !important;
+    border: none !important;
     box-shadow: none !important;
     cursor: pointer;
   }
 
   .Select-menu-outer,.Select-control,.Select-value-label,.Select-menu-outer,.Select-option {
-    background-color: $mm-brand-bg !important;
+    background-color: rgba(0,0,0,.1);
```
Comment (aliceriot) [inline static/scss/navbar.scss]: I think we should generally avoid inline colors and try to declare them all as variables - could you declare this in the `_variables.scss` file? (same for the other inline colors)

**Comment by aliceriot** [comment]
I do think the program selector is somewhat hard to read:

fiplle

what do you think?

**Comment by aliceriot** [comment]
I'm also wondering if the grey-on-black has enough contrast to meet accessibility guidelines, but I don't know if we have a quick way to test for that.

File: static/scss/navbar.scss (hunk 1 above)
Comment (roberthouse54) [inline static/scss/navbar.scss]: sure

**Comment by pdpinch** [comment]
You can plug the color numbers into the calculator at http://webaim.org/resources/contrastchecker/

We should aim for at least a WCAG AA pass. 

> I'm also wondering if the grey-on-black has enough contrast to meet accessibility guidelines, but I don't know if we have a quick way to test for that.

**Comment by roberthouse54** [comment]
It passes WCAG AA, but I will lighten it anyway

**Comment by roberthouse54** [comment]
I created variables for the colors, and lightened the font.

File: static/scss/navbar.scss (hunk 1 above)
Comment (aliceriot) [inline static/scss/navbar.scss]: there's still an inline `rgba` color here and on line 261 below

File: static/scss/navbar.scss [hunk 2]
```diff
@@ -83,19 +77,21 @@
     .links {
       display: flex;
       flex-direction: column;
-      border-bottom: thin solid $border-color;
+      border-bottom: thin solid rgba(255,255,255,.2);
```
Comment (aliceriot) [inline static/scss/navbar.scss]: also here

**Comment by aliceriot** [comment]
cool, my thought was that the dropdown for the program selector is a little bit too indistinguishable from the background. here's a screenshot without the chrome debugger overlay:

border

what do you think? I think it is a bit hard to visually distinguish the selector drop-down from the background, but I'm not sure how much of an issue this is / if it's just to my eye. My thoughts are that maybe using a contrasting border for the selector dropdown could make it more obvious? idk, haha.

**Comment by roberthouse54** [comment]
OK, I think I fixed all the things

**Pull request merged**